src/pages/index.astro

```diff
@@ -3,192 +3,152 @@ import Main from '../layouts/Main.astro'
 import OurWork from '../components/OurWork.astro'
 ---
 <Main title='Vivic Research'>
-    <div id="maincontent">
-        <div id="hero">
-            <h1>
-                today. tomorrow. together.
-            </h1>
-            <p>
-            Vivic Research is an economic consulting firm dedicated to empowering peers working toward social justice with data-driven research
-            </p>
-        </div>
-        <div class="maincenter" id="about">
-        </div>
-        <div class="maincenter" id="services">
-            <h2 class="center-title">Our Services</h2>
-            <div id="pillars">
-                <div class="pillar" id="pillar1">
-                    <div class="pillar-box" id="pink">
-                        Economic Research
-                    </div>
-                    <p>
-                        We develop boutique research studies focusing on targeted issues related to the intersection of
-                        policy, economics, and social justice. This includes but is not limited to literature reviews,
-                        policy recommendations, community consultation and cost/benefit analysis. Our past work has included
-                        costing an alternative municipal budget for the city of Ottawa with an emphasis on racial and gender
-                        equity, research on Employment Insurance and best practices for post-secondary education fees and
-                        debt cancellation.
-                    </p>
-                </div>
-                <div class="pillar" id="pillar2">
-                    <div class="pillar-box" id="white">
-                        Training
-                    </div>
-                    <p>
-                        We provide personalized training on quantitative research methods for teams, helping them build
-                        confidence and do their work more efficiently. Vivic offers SPSD/M training for beginners and more
-                        experienced users. Our introductory training "Getting Started: Policy Analysis Using SPSD/M" gets
-                        researchers quickly up-and-running with SPSD/M, saving time and frustration.
-                    </p>
-                </div>
-                <div class="pillar" id="pillar3">
-                    <div class="pillar-box" id="blue">
```

Comment:
I've gotten rid of the `.pillar-box`, `#pillar1`/`2/`3`, and color IDs, since they can be targeted by the `nth-child` selector more elegantly.